lib/api/admin/admin.test.ts

```diff
@@ -20,17 +20,31 @@ describe("Testing Admin Functions", () => {
           });
     })
 
+    describe("Testing register with an existing locker and a non-existent user", () => {
+        it("Should register a new locker using the provided name, email and locker", () => {
+            const registrations = register(1, 34, "Chicken Run", "Take@gmail.com");
+            expect(registrations).toBe(true);
+          });
+    })
+
+    describe("Testing register with a registered existing locker and a non-existent user", () => {
+        it("Should not register an already registered locker using the provided name, email and locker", () => {
+            const registrations = register(1, 34, "Darla Dupe", "Dish@gmail.com");
+            expect(registrations).toBe(false);
+          });
+    })
+
     describe("Testing getregistrations", () => {
         it("Returns all the registrations in the database", () => {
             const registration_list = getRegistrations();
-            expect(registration_list).toHaveLength(0);
+            expect(registration_list).toHaveLength(1);
           });
     })
 
     describe("Testing deregisterall", () => {
         it("Returns number of deregistered lockers", () => {
             const deregistered_list = deregisterAll();
-            expect(deregistered_list).toBe(0);
+            expect(deregistered_list).toBe(1);
           });
     })
 
```

Comment:
should test getRegistrations and deregisterAll with zero and multiple rows in the registrations table